spring-cloud-dataflow-rest-resource/src/main/java/org/springframework/cloud/dataflow/rest/util/DeploymentPropertiesUtils.java

```diff
@@ -233,6 +233,39 @@ public static void validateSkipperDeploymentProperties(Map<String, String> prope
 		return result;
 	}
 
+	/**
+	 * Retain all properties that are meant for the <em>deployer</em> of a given app
+	 * (those that start with {@code deployer.[appname]} or {@code deployer.*} or
+	 * {@code app.*}) and qualify all property values with the
```

Comment:
It only supports deployer.[appname] and deployer.*.  It does not support app.*.  will remove comment on merge.